### PR TITLE
leadership post rough draft

### PR DESCRIPTION
**Pull request by jessfraz**
cc @simpsoka

(obviously no rush, just putting a rough rough draft out there :) please rip it apart, etc

### CODE REVIEW

File: content/post/leadership-ci.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+in the best interest of everyone.
+
+The following are the sets of questions we brainstormed, but expect them to
+change over time as we experience and deal with new problems. These were
+started in [a gist](https://gist.github.com/simpsoka/14da775a63e22e5083141da5c48e6410)
+and are copied below. The diff of this post and the gist will serve as the
+evolution of this thought process.
+
+It's important to note that in some instances answering all the questions might
+take too much time. Perhaps prioritizing the most important ones in the moment
```
Comment (simpsoka) [inline content/post/leadership-ci.md:35]: Would be cool to make some suggestions about how to put this list into action. Maybe something like.... 

Answering all the questions may be a luxury at times, so we suggest breaking them down based on the situation you find yourself in: prioritize the most important ones to your role, have a few ‘go to’ questions, or categorize them based on the situations you find yourself in more often. The important part of this list is to help us navigate a difficult situation while still maintaining the integrity we intend for ourselves as leaders.

File: content/post/leadership-ci.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+> ― Malcolm Gladwell, Blink: The Power of Thinking Without Thinking
+
+
+As leaders, being consistent in our actions even when under pressure is of
+utmost importance. When writing and delivering software we rely on our
+continuous integration (CI) infrastructure and test suites to tell us when a test is
+failing and code should not be merged.
+
+As leaders, before acting or making decisions it would be nice to have a set of
+tests and checks to make sure we are in the right headspace to think and behave
```
Comment (simpsoka) [inline content/post/leadership-ci.md]: Tests and checks, established ahead of time, to make sure we are in the right headspace to think, behave and make decisions that are in the best interest of everyone and our company.

File: content/post/leadership-ci.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+> “understanding the true nature of instinctive decision making requires us to be forgiving of those people trapped in circumstances where good judgment is imperiled.” 
+> ― Malcolm Gladwell, Blink: The Power of Thinking Without Thinking
+
+
+As leaders, being consistent in our actions even when under pressure is of
+utmost importance. When writing and delivering software we rely on our
+continuous integration (CI) infrastructure and test suites to tell us when a test is
+failing and code should not be merged.
+
```
Comment (simpsoka) [inline content/post/leadership-ci.md:19]: Hmmm, consistency isn’t always good unless it’s consistent integrity..... we want to have room to grow and change our minds - there’s a question for this, too. So maybe something that gives leaders a little room, while still suggesting a structure to move around in. 

Maybe like..... 

As leaders, setting up a structure that helps us navigate decisions under pressure is of the utmost importance.

File: content/post/leadership-ci.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    **Fail:** I’m intentionally using words that aren’t representative of what I’m trying to communicate.
+
+14. Am I speaking for people or letting them speak for themselves?
+
+    **Pass:** I am doing the minority of the speaking and people are disagreeing
+with my opinions.
+
+    **Fail:** I am being quoted back to myself. I am talking the majority of the
+time.
```
Comment (simpsoka) [inline content/post/leadership-ci.md:130]: Maybe add another callout to follow along in the gist if people want to see how this list evolves - how we prioritize it... etc.

**Review by simpsoka — COMMENTED** [review]
Left you a few suggestions up for grabs. :D

File: content/post/leadership-ci.md (hunk 1 above)
Comment (jessfraz) [reply content/post/leadership-ci.md:35]: oooo I love this :)

**Comment by jessfraz** [comment]
awesome! I pushed the changes :)

File: content/post/leadership-ci.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+As leaders, setting up a structure that helps us navigate decisions under
+pressure is of the utmost importance. 
+When writing and delivering software we rely on our
+continuous integration (CI) infrastructure and test suites to tell us when a test is
+failing and code should not be merged.
+
+As leaders, before acting or making decisions it would be nice to have a set of
+tests and checks, established ahead of time, to make sure we are in the
+right headspace to think, behave and make decisions that are in the best 
+interest of everyone and our company. There are devastating consequences to
```
Comment (jessfraz) [inline content/post/leadership-ci.md:23]: Added this sentence, hope it makes sense :) "There are devastating consequences to taking actions based on fear and pride; we hope this set of questions enables taking action based on growth, humility, inclusion, and soulful reflection."

**Comment by jessfraz** [comment]
merged :)

**Pull request closed without merging**